data/hails.txt

```diff
@@ -1056,10 +1120,15 @@ phrase "friendly navy"
 		" "
 	word
 		"you."
+		"you and your family."
+		"the people."
+		"the civilians."
```

Comment:
Play with the combinations in your mind. These three additions don't work very well with many of the existing strings.

Reply:
a good trick is to try to make weird sentences, if they are still correct it should be fine